src/FunctionApps/python/tests/IntakePipeline/conversion_test.py

```diff
@@ -107,59 +107,10 @@ def test_convert_message_to_fhir_success(mock_fhir_post):
     assert response == {"hello": "world"}
 
 
-@mock.patch("requests.post")
-def test_convert_message_to_fhir_failure(mock_fhir_post):
-    mock_fhir_post.return_value = mock.Mock(
-        status_code=400, json=lambda: {"hello": "world"}
-    )
-
-    message = "MSH|blah|foo|test\nPID|some^text|blah\nOBX|foo||||bar^baz&foobar"
-    response = convert_message_to_fhir(
-        message,
-        "some-filename-0",
-        "Hl7v2",
-        "VXU_V04",
-        "microsofthealth/fhirconverter:default",
-        "some-token",
-        "some-fhir-url",
-    )
-
-    mock_fhir_post.assert_called_with(
-        url="some-fhir-url/$convert-data",
-        json={
-            "resourceType": "Parameters",
-            "parameter": [
-                {"name": "inputData", "valueString": message},
-                {"name": "inputDataType", "valueString": "Hl7v2"},
-                {
-                    "name": "templateCollectionReference",
-                    "valueString": "microsofthealth/fhirconverter:default",
-                },
-                {"name": "rootTemplate", "valueString": "VXU_V04"},
-            ],
-        },
-        headers={"Authorization": "Bearer some-token"},
-    )
-
-    assert response == {}
-
-
 @mock.patch("requests.post")
 @mock.patch("logging.error")
-def test_log_fhir_operationoutcome(mock_log, mock_fhir_post):
-    mock_fhir_post.return_value = mock.Mock(
-        status_code=400,
-        json=lambda: {
-            "resourceType": "OperationOutcome",
-            "issue": [
-                {
-                    "severity": "fatal",
-                    "code": "code-invalid",
-                    "diagnostic": "error-description",
-                }
-            ],
-        },
-    )
+def test_log_generic_error(mock_log, mock_fhir_post):
```

Comment:
I think this test is conflating two things, both checking that the logging was properly called while also asserting that the return value from `convert_message_to_fhir` is correct. I'd like to see this split back out so that the tests are more explicit.